cypress/objects/loginPage.js

```diff
@@ -27,12 +27,25 @@ getRole(){
 return cy.get('[id="loginForm.role"]');
 }
 
-roleLabel(){
+roleLabeluser(){
```

Comment:
Do here what you did with getUserOption(): insert a parameter to this function:

> roleLabeluser(roleName){
>     return cy.get(`[aria-label="${roleName}"]`);
> }

And then you won't need the other 4 functions, since this one will be universal :)